### PR TITLE
feat(alertmanager): add custom headers to alert manager

### PR DESCRIPTION
**Pull request by Umaaz**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area config

/area outputs

> /area tests

**What this PR does / why we need it**:

Add support for custom headers when using the alertmanager output.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by Umaaz** [comment]
I think I got everything for this. Let me know if I missed anything.

**Review by Issif — COMMENTED** [review]
Can you also update the docs please https://github.com/falcosecurity/falcosidekick/blob/master/docs/outputs/alertmanager.md

File: docs/outputs/alertmanager.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 ## Configuration
 
 | Setting                                 | Env var                                 | Default value                                                        | Description                                                                                                                                                                                                                              |
-| --------------------------------------- | --------------------------------------- | -------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|-----------------------------------------|-----------------------------------------|----------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (Umaaz) [inline docs/outputs/alertmanager.md:17]:
```suggestion
| --------------------------------------- | --------------------------------------- | -------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

**Comment by Umaaz** [comment]
Update the docs based on the grafana version.

**Review by Issif — APPROVED** [review]

**Pull request merged**